paparazzi/src/main/java/app/cash/paparazzi/HtmlReportWriter.kt

```diff
@@ -57,7 +57,7 @@ import javax.imageio.ImageIO
  * paparazzi.js
  * ```
  */
-internal class HtmlReportWriter(
+class HtmlReportWriter @JvmOverloads constructor(
```

Comment:
Does this class need to be in the public API?

Reply:
Good question.  In the referenced issue, the snapshot handler is the second-to-last arg, so the equivalent java overload makes it required to set this when setting RenderExtensions.

I _could_ swap the parameter ordering, but wasn't sure if that was good form, i.e., does that mean all future parameter additions need to not be last?

Reply:
Interesting. For Java we'd probably just want a separate `Builder`..

Reply:
Ah true, didn't even think of providing _that_ level of support.  Isn't everyone on Kotlin by now?  😛 

I'll merge this for now and follow-up with Builder.

Reply:
We can maybe just say "sorry no explicit Java support"? IDK